src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml.cpp

```diff
@@ -0,0 +1,282 @@
+    {
+        switch (ViewModel->Parity)
+        {
+        case 0:
+            Parity = m_resourceLoader->GetString(L"KGFParityUnknown");
+            break;
+        case 1:
+            Parity = m_resourceLoader->GetString(L"KGFParityOdd");
+            break;
+        case 2:
+            Parity = m_resourceLoader->GetString(L"KGFParityEven");
+            break;
+        case 3:
+            Parity = m_resourceLoader->GetString(L"KGFParityNeither");
+            break;
+        default:
+            Parity = m_resourceLoader->GetString(L"KGFParityError");
+        }
+    }
+}
+
+void KeyGraphFeaturesPanel::SetPeriodicityStringProperty()
+{
+    if (ViewModel->Periodicity->Size != 0)
+    {
+        auto periodicityType = ViewModel->Periodicity->First()->Current->Key->Data();
+        switch (*periodicityType)
+        {
+        case '0':
+            Periodicity = m_resourceLoader->GetString(L"KGFPeriodicityUnknown");
+            break;
+        case '1':
+            Periodicity = ViewModel->Periodicity->First()->Current->Value;
+            break;
+        case '2':
+            Periodicity = m_resourceLoader->GetString(L"KGFPeriodicityNotPeriodic");
+            break;
+        default:
+            Periodicity = m_resourceLoader->GetString(L"KGFPeriodicityError");
+        }
+    }
+}
+
+void KeyGraphFeaturesPanel::SetMonotoncityStringProperty()
+{
+    if (ViewModel->Monotonicity->Size != 0)
+    {
+        int monotonicitySize = ViewModel->Monotonicity->Size;
+        int i = 0;
+        for (auto item : ViewModel->Monotonicity)
+        {
+            Monotonicity += item->Key;
+            Monotonicity += " ";
+
+            auto monotonicityType = item->Value->Data();
+            switch (*monotonicityType)
+            {
+            case '0':
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityUnknown");
+                break;
+            case '1':
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityIncreasing");
+                break;
+            case '2':
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityDecreasing");
+                break;
+            case '3':
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityConstant");
+                break;
+            default:
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityError");
+                break;
+            }
+
+            if (i < monotonicitySize - 1)
+            {
+                Monotonicity += "\n";
+            }
+        }
+    }
+}
+
+String ^ KeyGraphFeaturesPanel::ConvertVectorToString(Windows::Foundation::Collections::IObservableVector<String ^> ^ inVector)
+{
+    String ^ outString = ref new String();
+    int vectorSize = inVector->Size;
+    for (int i = 0; i < vectorSize; i++)
+    {
+        outString += inVector->GetAt(i);
+        if (i < vectorSize - 1)
+        {
+            outString += "\n";
```

Comment:
In this case since I am using a new line, it will be ok. Using a ListView with MathRichEditBoxes will be much more expensive than using the new line character.